components/omega/doc/design/OmegaV1GoverningEqns.md

```diff
@@ -0,0 +1,765 @@
+
+#### Final Continuous Equations
+
+The final form of the continuous conservation equations for a non-Boussinesq, hydrostatic ocean are
+
+momentum:
+
+$$
+\frac{\partial {\bf u}}{\partial t}
++ \zeta {\bf u}^\perp + f {\bf u}^\perp
++ w\frac{\partial {\bf u}}{\partial z}
+  = - \frac{1}{\rho} \nabla_z p
+  - \nabla_z K
++ {\bf D}^u + {\bf F}^u
+$$ (continuous-momentum-final)
+
+mass:
+
+$$
+\frac{\partial \rho }{\partial t}
+ + \nabla_z \cdot \left( \rho  {\bf u}  \right)
+ + \frac{\partial}{\partial z} \left(  \rho  w \right)
+= 0
+$$ (continuous-mass-final)
+
+tracers:
+
+$$
+\frac{\partial \rho \varphi}{\partial t}
+ + \nabla_z \cdot \left( \rho \varphi {\bf u}  \right)
+ + \frac{\partial}{\partial z} \left(  \rho \varphi w \right)
+= D^\varphi + Q^\varphi
+$$ (continuous-tracer-final)
+
+equation of state:
+
+$$
+\rho = f_{eos}(p,\Theta,S).
+$$ (continuous-eos-final)
+
+hydrostatic pressure:
+
+$$
+p(x,y,z) = p^{surf}(x,y) + \int_{z}^{z^{surf}} \rho g dz'.
+$$ (continuous-hydrostatic-pressure-final)
+
+Here the $\nabla$ operators are exactly horizontal; we expand terms for tilted layers in the following section. The momentum diffusion terms include Laplacian (del2), biharmonic (del4), and vertical viscosity,
+
+$$
+{\bf D}^u
+= \nu_2 \nabla^2 {\bf u} - \nu_4 \nabla^4 {\bf u}
+ + \frac{\partial }{\partial z} \left( \nu_v \frac{\partial {\bf u}}{\partial z} \right)
+$$ (continuous-h_mom_diff)
+and may also include a Rayleigh drag and eventually parameterizations. Momentum forcing is due to wind stress and bottom drag. Similarly, the tracer diffusion terms include Laplacian (del2), and vertical viscosity,
+
+$$
+D^\varphi =
+ \nabla\cdot\left(\rho \kappa_2 \nabla\varphi \right)
++ \rho \frac{\partial }{\partial z}
+  \left( \kappa_v \frac{\partial \varphi}{\partial z} \right),
+$$ (continuous-v_tr_diff)
+
+and may also include a biharmonic (del4) term and parameterizations such as Redi mixing. Sources and sinks include surface fluxes from the atmosphere and land, and bio-geo-chemical reactions.
+All of the diffusion and forcing terms are written in more detail with the [Discrete Equations](#discrete-equations) below.
+
+## 4. Layered Equations
```

Comment:
I would like to expand this section to include the equivalent derivation to Ringler et al. (2013) A2 but with mass-thickness rather than geometric thickness.  I'm going to finish my review for now so I can work on that.